index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <main>
+      <header class="header-box">
+        <a href="#">Welcome</a>
+        <img src="/images/Menu-icon.png" alt="Menu icon" />
+      </header>
```

Comment:
- [ ] Congratulations on implementing all the other changes! But it seems that the menu image is still not wrapped inside a nav tag as requested in the previous review, kindly fix that
![image](https://user-images.githubusercontent.com/73140461/148459467-ef44c933-835d-49ff-b918-70a10eba38df.png)